test/ffmpeg/ffmpeg_native_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule VideoCompositor.FFmpeg.Native.Test do
+  use ExUnit.Case, async: true
+
+  alias Membrane.RawVideo
+  alias Membrane.VideoCompositor.FFmpeg.Native
+  alias Membrane.VideoCompositor.Test.Utility
+
+  @tag :tmp_dir
+  test "compose doubled raw video frames on top of each other", %{tmp_dir: tmp_dir} do
```

Comment:
Something's not well configured, because `tmp_dir` is included in git with the output files.